src/python/pants/backend/python/goals/setup_py_test.py

```diff
@@ -226,14 +226,14 @@ def test_generate_chroot(chroot_rule_runner: RuleRunner) -> None:
 
 
 def test_invalid_binary(chroot_rule_runner: RuleRunner) -> None:
-    chroot_rule_runner.create_file("src/python/invalid_binary/app.py")
+    chroot_rule_runner.create_files("src/python/invalid_binary", ["app1.py", "app2.py"])
     chroot_rule_runner.add_to_build_file(
         "src/python/invalid_binary",
         textwrap.dedent(
             """
             python_library(name='not_a_binary', sources=[])
-            pex_binary(name='no_explicit_entrypoint', sources=["app.py"])
-            pex_binary(name='malformed_entrypoint', entry_point='invalid_binary.app')
+            pex_binary(name='no_explicit_entrypoint', sources=["app1.py"])
+            pex_binary(name='malformed_entrypoint', entry_point='invalid_binary.app2')
```

Comment:
Without this, we have an issue that `:malformed_entrypoint` infers a dep on `:no_explicit_entrypoint`, which causes setup.py's Ambiguous Owners error. This seemed like the simplest fix.

It won't be relevant once the `sources` field is removed.